docs/releases/5.0.8.txt

```diff
@@ -23,3 +23,7 @@ Bugfixes
 * Fixed a regression in Django 5.0.7 that caused a crash in
   ``LocaleMiddleware`` when processing a language code over 500 characters
   (:ticket:`35627`).
+
+* Fixed a bug in Django 5.0 which caused ``Model.validate_constraints()`` to
+  incorrectly raise validation errors for constraints referring to fields using
+  ``Field.db_default`` (:ticket:`35638`).
```

Comment:
The issue was more wide spread than that as `validate_unique` which powers `Field.unique=True`, `Meta.unique_together`, and `Date(Time?)Field.unique_for_(date, month, year)` was also affected. Maybe something along the lines of 

```suggestion
* Fixed a bug in Django 5.0 which caused constraint validation to either crash
  or incorrectly raise validation errors for constraints referring to fields using
  ``Field.db_default`` (:ticket:`35638`).
```